src/main/common/colorconversion.c

```diff
@@ -88,3 +88,16 @@ rgbColor24bpp_t hsvToRgb24(const hsvColor_t* c)
     return r;
 }
 
+rgbwColor32bpp_t hsvToRgbw32(const hsvColor_t* c)
+{
+    rgbwColor32bpp_t r;
+    rgbColor24bpp_t rgb = hsvToRgb24(c);
+
+    r.rgbw.r = rgb.rgb.r;
+    r.rgbw.g = rgb.rgb.g;
+    r.rgbw.b = rgb.rgb.b;
+    r.rgbw.w = c->w;
```

Comment:
this is the source of the issue with 'w' being added to hsvColor_t.

I feel that perhaps a suitable algorithm i should be used here when sets rgbw.w correctly based on just the hsv values.